### DISCUSSION TITLE
Is it possible to build public API endpoints with React-Remix for shopify app.

### DISCUSSION THREAD

**Discussion by chirag-08x** [remix-run/remix#10326] [Q&A]
So I am currently building a shopify app with react-remix, and inside that project, I want to create a Public API endpoint, which can be called from external sites/resources, that would just return the list of all the products from the store, on which the app is installed. on Also with the API call we will also pass a Bearer token to check if the API call is authorized or not.

Please let me know if that's possible, I have written the code for it, but I am unable to call the API, I can't figure out using which URL should I call it upon.

**Comment by sergiodxa** [comment]
Yes, here are some docs on [API routes](https://remix.run/docs/en/main/guides/api-routes) and [resource routes](https://remix.run/docs/en/main/guides/resource-routes), those also works on [RR](https://reactrouter.com/how-to/resource-routes).
